src/argilla/client/users.py

```diff
@@ -278,10 +282,10 @@ def me(cls) -> "User":
             >>> from argilla import rg
             >>> user = rg.User.me()
         """
-        client = cls.__active_client(httpx=False)
+        client = cls.__active_client()
         try:
-            user = users_api.whoami(client)  # .parsed
-            return cls.__new_instance(client.httpx, user)
+            user = users_api.whoami(client).parsed
+            return cls.__new_instance(client, user)
```

Comment:
Same here, I would revert this until fixed

Reply:
rolled back